src/GraphQLParser.ApiTests/ApiApprovalTests.Public_Api_Should_Not_Change_Inadvertently.approved.txt

```diff
@@ -308,6 +317,10 @@ namespace GraphQLParser.AST
     {
         System.Collections.Generic.List<GraphQLParser.AST.GraphQLDirective>? Directives { get; set; }
     }
+    public interface IHaveDescription
+    {
+        GraphQLParser.AST.GraphQLDescription? Description { get; set; }
```

Comment:
As I understand it, the same elements of the schema can have descriptions, which can have comments, that is, all of them. So I propose to move this property into `ASTNode` and remove interface.

Reply:
Not all nodes can have a description according to the specification. 
See http://spec.graphql.org/draft/#sec-Document-Syntax, search for "Description". I listed all nodes which are allowed to have the description.

It is reasonable why spec designers decided to do so: there is no sense to provide documentation in the query, so only TypeSystemDefinitions should have the descriptions. Even type extensions don't have a description, I assume it's because they extend existing type and thus derive the description (documentation) from them.

Reply:
OK